config/db/schemas

```diff
@@ -151,11 +152,17 @@ INSERT INTO reference_event_types (id, description) values (2, 'Audits');
 INSERT INTO reference_event_types (id, description) values (3, 'Reports');
 
 CREATE TABLE "reference_event_subtypes" (
-  "id" integer NOT NULL PRIMARY KEY AUTOINCREMENT,
-  "description" varchar(100) NOT NULL
+  "event_type_id" integer NOT NULL,
+  "event_subtype_id" integer NOT NULL,
```

Comment:
Also please change it in `FOREIGN KEY(event_subtype_id) REFERENCES reference_event_subtypes(id)` of  `event_notification_triggers` table.